### PR TITLE
Support artifact's path in Build-Info

### PR DESCRIPTION
**Pull request by barbelity**
- [ ] I signed [JFrog's CLA](https://secure.echosign.com/public/hostedForm?formid=5IYKLZ2RXB543N).
- [ ] All [tests](https://github.com/jfrog/jfrog-cli#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [ ] This pull request is on the dev branch.
- [ ] I used gofmt for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: artifactory/utils/docker/buildinfo.go [hunk 1]
```diff
@@ -293,7 +293,7 @@ func getManifest(imageId string, searchResults map[string]utils.ResultItem, serv
 		return nil, buildinfo.Artifact{}, buildinfo.Dependency{}, errorutils.CheckError(errors.New("Found incorrect manifest.json file, expecting image ID: " + imageId))
 	}
 
-	artifact = buildinfo.Artifact{Name: "manifest.json", Type: "json", Checksum: &buildinfo.Checksum{Sha1: item.Actual_Sha1, Md5: item.Actual_Md5}}
+	artifact = buildinfo.Artifact{Name: "manifest.json", Type: "json", Checksum: &buildinfo.Checksum{Sha1: item.Actual_Sha1, Md5: item.Actual_Md5}, Path: item.Repo + "/" + item.Path + "/" + item.Name}
```
Comment (eyalbe4) [inline artifactory/utils/docker/buildinfo.go:296]: You can use path.join here.

File: artifactory/utils/docker/buildinfo.go [hunk 2]
```diff
@@ -321,7 +321,7 @@ func getConfigLayer(imageId string, searchResults map[string]utils.ResultItem, s
 		return nil, buildinfo.Artifact{}, buildinfo.Dependency{}, err
 	}
 
-	artifact = buildinfo.Artifact{Name: digestToLayer(imageId), Checksum: &buildinfo.Checksum{Sha1: item.Actual_Sha1, Md5: item.Actual_Md5}}
+	artifact = buildinfo.Artifact{Name: digestToLayer(imageId), Checksum: &buildinfo.Checksum{Sha1: item.Actual_Sha1, Md5: item.Actual_Md5}, Path: item.Repo + "/" + item.Path + "/" + item.Name}
```
Comment (eyalbe4) [inline artifactory/utils/docker/buildinfo.go:324]: You can use path.join here.

File: go.mod [hunk 3]
```diff
@@ -25,7 +25,7 @@ require (
 	gopkg.in/yaml.v2 v2.2.2
 )
 
-//replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go dev
+replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v0.10.0
```
Comment (eyalbe4) [inline go.mod:28]: I think you want to use the dev branch here, to get the latest additions to jfrog-client-go.

Comment (barbelity) [reply go.mod:28]: For some reason, every time I change it to 'dev' IntelliJ overrides it to this specific version.

**Pull request merged**